src/api/base/index.ts

```diff
@@ -48,19 +48,46 @@ export default class BaseApi {
     }
   }
 
-  protected post(url: string, data: unknown, options: Options = {}): Promise<Response> {
-    return this.handleMethod(url, 'POST', this.fetch.bind(this), this.API_URL_GETTER, options, data)
+  protected post(
+    url: string,
+    data: unknown,
+    options: Options = {},
+    fetchFn?: 'singleEnv' | 'multipleEnvs'
```

Comment:
if there's two values anyways, why not making a boolean with a default value. `postInAllEnvironments = true`